### DISCUSSION TITLE
Quick Start but 404 Not Found: .../apache/incubator-amoro/.../demo-cluster.sh

### DISCUSSION THREAD

**Discussion by oliverxt** [apache/amoro#2840] [Q&A]
To [setup](https://amoro.apache.org/quickstart-setup/#setup-from-docker-composee) for a [quick start](https://amoro.apache.org/quickstart-setup/#setup-from-docker-composee), I need to `wget https://raw.githubusercontent.com/apache/incubator-amoro/master/docker/demo-cluster.sh
`. However, it leads to a 404 error. 
I have also searched `incubator-amoro` in _apache_ repository, but I couldn't find it either.

**Comment by zhoujinsong** [comment]
Thanks for reporting this.

We are redesigning the quick start process and related docker images, you can find more information here: https://lists.apache.org/thread/4dq0q1kwkny6x4jwyonnrqz985jlgdov
So some of the scripts may be dropped in the master branch.

You can still find them in old version branches like : https://github.com/apache/amoro/blob/0.6.x/docker/demo-cluster.sh
